tests/test_skills/test_abstract_round_abci/test_base.py

```diff
@@ -939,7 +939,9 @@ def test_check_majority_possible_passes_when_vote_set_nonempty_and_check_passes(
         - the threshold is 2
         - the other voter can vote for the same item of the first voter
         """
-        AbstractRound.check_majority_possible({"voter": "item"}, 2)
+        votes = {"alice": DummyPayload("alice", True)}
+        votes_by_participants: Dict[str, BaseTxPayload] = votes
+        AbstractRound.check_majority_possible(votes_by_participants, 2)
```

Comment:
here and below I have changed and payload that was a string into a `DummyPayload`